### PR TITLE
Moves electron-app to frontend/app

### PR DESCRIPTION
**Pull request by kelsos**
Restructuring in preparation for extracting components in the component library and moving the dev-proxy here

### CODE REVIEW

**Comment by kelsos** [comment]
any pr that needs to be rebased on top of that and changes the frontend would have to also do a `git mv electron-app/* frontend/app` (not sure about the exact syntax) to move the new files to the new structure @isidorosp

File: docs/installation_guide.rst [hunk 1]
```diff
@@ -302,10 +302,10 @@ Installing Electron and Running Rotki
 
 1. In your terminal, navigate to your rotki development directory and enter the following commands to install electron and its dependencies::
 
-    cd electron-app
+    cd frontend/app
```
Comment (isidorosp) [inline docs/installation_guide.rst:305]:
```suggestion
    cd frontend\app
```

Comment (isidorosp) [reply docs/installation_guide.rst:305]: Windows slash

File: docs/installation_guide.rst [hunk 2]
```diff
@@ -302,10 +302,10 @@ Installing Electron and Running Rotki
...
     npm ci
 
-2. At this point, your terminal's cwd should be ``<rotki development directory>\electron-app\`` and the rotki virtualenv should be activated. You should now be able to start rotki in development mode by executing::
+2. At this point, your terminal's cwd should be ``<rotki development directory>\frontend/app\`` and the rotki virtualenv should be activated. You should now be able to start rotki in development mode by executing::
```
Comment (isidorosp) [inline docs/installation_guide.rst:308]:
```suggestion
2. At this point, your terminal's cwd should be ``<rotki development directory>\frontend\app\`` and the rotki virtualenv should be activated. You should now be able to start rotki in development mode by executing::
```
Windows slash

File: package.bat [hunk 3]
```diff
@@ -67,7 +67,7 @@ if %errorlevel% neq 0 (
 )
 
 @echo on
-CD electron-app/
+CD frontend/app/
```
Comment (isidorosp) [inline package.bat:70]:
```suggestion
CD frontend\app\
```
Windows slash (both actually work but might as well keep it consistent)

**Pull request merged**